src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+
+type Props = {
+  button: string,
+  label: string,
+  items: Array<Object>
```

Comment:
`$ReadOnlyArray<{value: string, label: string}>`
1. this will be immutable hence 'read only'
2. we still need to specify what kind of objects does it include